src/components/Supporters/index.jsx

```diff
@@ -0,0 +1,227 @@
+        width: 250,
+    },
+    {
+        name: "Voltron Data",
+        link: "https://voltrondata.com/",
+        light: "img/supporters/voltron_light.svg",
+        dark: "img/supporters/voltron_dark.svg",
+        width: 250,
+    },
+    {
+        name: "Quansight Labs",
+        link: "https://labs.quansight.org/",
+        light: "img/supporters/quansight_light.svg",
+        dark: "img/supporters/quansight_dark.svg",
+        width: 250,
+    },
+    {
+        name: "QuantStack",
+        link: "https://quantstack.net/",
+        light: "img/supporters/quantstack_light.svg",
+        dark: "img/supporters/quantstack_dark.svg",
+        width: 250,
+    },
+];
+
+export default function Supporters() {
+    return (
+        <div className={[styles.supporters, styles.section_padding].join(" ")}>
+            <div className={styles.supporters_conda_forge}>
+                <h1>Supporters</h1>
+                <p>
+                    If you like conda-forge and want to support our mission,
+                    please consider making a{" "}
+                    <Link to="https://opencollective.com/conda-forge">
+                        donation
+                    </Link>{" "}
+                    to support our efforts.
+                </p>
+            </div>
+            <div className={styles.fiscal_sponsor}>
+                <div className={styles.numfocus_card}>
+                    <Link to="https://numfocus.org/">
+                        <ThemedImage
+                            className={styles.image}
+                            alt="NumFOCUS Logo"
+                            sources={{
+                                light: useBaseUrl(
+                                    "/img/supporters/numfocus.svg"
+                                ),
+                                dark: useBaseUrl(
+                                    "/img/supporters/numfocus.svg"
+                                ),
+                            }}
+                            width="100%"
+                        />
+                    </Link>
+                </div>
+                <div className={styles.about_numfocus}>
+                    <h3>
+                        conda-forge is a{" "}
+                        <span className="gradient_text">
+                            fiscally sponsored
+                        </span>{" "}
+                        project of NumFOCUS.
+                    </h3>
+                    <p>
+                        A nonprofit dedicated to supporting the open source
+                        scientific computing community.
+                    </p>
+                </div>
+            </div>
+            <div className={styles.other_supporters}>
+                <div className={styles.supporters_card}>
+                    <div className={styles.supporters_conda_forge}>
+                        <h2>
+                            <span className="gradient_text">Financial</span>{" "}

```

Comment:
It looks like JS/CSS naming conventions prefer `camelCase` instead of `snake_case`, but your class name here is `snake_case`. Would it make sense to standardize this?

Reply:
It depends from org to org. I have seen many orgs using other cases as well such as `PasalCase` and `kebab-case`. In this project I have used `snake_case` for better readability.